locales/ca_ES/LC_MESSAGES/duckduckgo.po

```diff
@@ -6078,7 +6078,7 @@ msgstr "Unitats de mesura"
 #. A tooltip that is shown after the user has clicked the swap units button. Example: https://duckduckgo.com/?q=5+meters+to+cm&ia=answer
 msgctxt "Conversions module"
 msgid "Units swapped"
-msgstr ""
+msgstr "Divises canviades"
```

Comment:
Not that this is not "currency swapped", but "units". Please check [this example](https://duckduckgo.com/?q=5+meters+to+cm&ia=answer). It is used for measurement units.